pkgs/applications/editors/rehex/default.nix

```diff
@@ -0,0 +1,39 @@
+{ capstone
+, fetchFromGitHub
+, jansson
+, lib
+, stdenv
+, wxGTK30
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rehex";
+  version = "0.3.1";
+
+  buildInputs = [
```

Comment:
Nit pick: this is normally put behind the source fetch.

Reply:
Done :grin: